spec/requests/posts_controller_spec.rb

```diff
@@ -1852,6 +1852,17 @@
             I18n.t("activerecord.errors.models.topic.attributes.base.unable_to_unlist"),
           )
         end
+
+        context "with apply_modifier" do
+          it "allows mentioning any groups" do
+            plugin = Plugin::Instance.new
+            modifier = :mentionable_groups
+            proc = Proc.new { Group.all }
+            DiscoursePluginRegistry.register_modifier(plugin, modifier, &proc)
+
+            expect(Group.mentionable(user)).to eq(Group.all)
```

Comment:
@rngus2344 isn't there some cleanup needed here? I thought you had to "unregister" these? do other modifier specs have cleanup?

Reply:
Oh yes, there was. I will add that real quick